styles.css

```diff
@@ -85,4 +87,185 @@
   .social-icons li {
     margin: 0 10px;
   }
+
+  .list-item {
+    background-color: #ebf0ee;
+    border: 1px solid #d0d9d4;
+    border-radius: 5px;
+    width: 90%;
+    margin-top: 10px;
+  }
+
+  .first-item {
+    border-radius: 8px 8px 5px 5px;
+    margin-top: 200px;
+    background-color: white;
+    width: 100%;
```

Comment:
- [ ] Kindly make sure you use `CSS Grid` in the main container that contains all of the cards in the `work` section as stated in the project requirements 🙏